lib/matplotlib/image.py

```diff
@@ -354,6 +372,20 @@ def _make_image(self, A, in_bbox, out_bbox, clip_bbox, magnification=1.0,
             out_height = int(out_height_base)
         out_shape = (out_height, out_width)
 
+        # decide if we are going to do anti-aliasing or not on the image
```

Comment:
do you also need to do the check here?  isn't it redundant with the one in _resample above?

Reply:
No, didn't catch this.   You refactored this, so I now have that logic where you refactored.